syntax = "proto3";
package proto;

message Instruction {
    bytes uuid = 1;
    uint64 ea = 2;
    int64 number_of_uses = 3;
    bool is_fallthrough = 4;
    bool is_pei = 5;
}
